### PR TITLE
Font changes

### PR DESCRIPTION
**Pull request by roguemacaroon**
Added classes for fonts, and MQ for mobile fonts. 
Took out contact us from footer, removed VH from hero section.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -106,12 +107,13 @@ img {
...
 /* Text */
 
-h1 {
+.main-headings {
 	font-family: var(--main-heading-font);
 	font-size: var(--font-xxl);
 }
 
-h2 {
+.sub-headings {
```
Comment (maxharrisnet) [inline style.css]: You can probably target `h2` instead of adding this class to each one.

File: style.css [hunk 2]
```diff
@@ -106,12 +107,13 @@ img {
 
 /* Text */
 
-h1 {
+.main-headings {
```
Comment (maxharrisnet) [inline style.css]: H1?

File: index.html [hunk 3]
```diff
@@ -64,7 +64,7 @@ <h1>Welcome To Mother Tongues Network</h1>
     <section class="homepage-content">
       <img src="https://placehold.co/600x400" alt="" />
       <div class="home-text">
-        <h2>Mother Tongues Network (MTN)</h2>
+        <h2 class="sub-headings">Mother Tongues Network (MTN)</h2>
```
Comment (maxharrisnet) [inline index.html]: You can probably target the H2 instead of adding all of these. We also need the styles on About and Contact.

File: style.css [hunk 4]
```diff
@@ -106,12 +107,13 @@ img {
...
 	font-family: var(--main-heading-font);
 	font-size: var(--font-xxl);
 }
 
-h2 {
+.sub-headings {
+	font-family: var(--sub-heading-font);
 	font-size: var(--font-lg);
 }
```
Comment (maxharrisnet) [inline style.css:119]: Can we edit the bottom margin? Right now it's the browser default and it looks like too much space.

File: style.css [hunk 5]
```diff
@@ -106,12 +107,13 @@ img {
```
Comment (maxharrisnet) [inline style.css]: Can we please add the line-heights as well? The paragraphs look too tight.

File: style.css [hunk 6]
```diff
@@ -191,7 +193,8 @@ header {
 	align-items: center;
 }
```
Comment (maxharrisnet) [inline style.css:195]: Don't forget to apply the fonts to the form inputs and buttons. Right now these are being set to sans-serif by normalize.css.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
@roguemacaroon I just added some feedback, please take a look.

File: style.css (hunk 2 above)
Comment (roguemacaroon) [reply style.css]: for accessibility h1 is used in the header for the logo text and then used again in the hero, so changing this one breaks both.

File: index.html (hunk 3 above)
Comment (roguemacaroon) [reply index.html]: changed this to h2 only and removed class

File: style.css (hunk 4 above)
Comment (roguemacaroon) [reply style.css:119]: added margin-bottom 0

File: style.css (hunk 5 above)
Comment (roguemacaroon) [reply style.css]: added a variable of 35px line height and applied to paragraphs

File: style.css (hunk 6 above)
Comment (roguemacaroon) [reply style.css:195]: applied --body-font to email form

File: style.css (hunk 1 above)
Comment (roguemacaroon) [reply style.css]: done

File: style.css (hunk 2 above)
Comment (roguemacaroon) [reply style.css]: changed logo text to span and implemented h1 instead of classes.

**Comment by roguemacaroon** [comment]
Made changes based on feedback, also removed some redundant code

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
Looks good, thanks!

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**